Pre-installed Tez on Dataproc 1.3 is failing when example job is submitted.

Steps to reproduce:
1. Create dataproc 1.3 cluster: 
`gcloud dataproc clusters create tez --image-version 1.3 `
2. Ssh into master node:
`gcloud compute ssh tez-m`
3. Run example Tez job:
`hadoop jar /usr/lib/tez/tez-examples.jar orderedwordcount file:///usr/lib/tez/LICENSE-MIT /tmp/tez-out`

Output: 

> java.lang.NoClassDefFoundError: org/apache/tez/dag/api/TezConfiguration
> 	at java.lang.Class.getDeclaredMethods0(Native Method)
> 	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
> 	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
> 	at java.lang.Class.getMethod0(Class.java:3018)
> 	at java.lang.Class.getMethod(Class.java:1784)
> 	at org.apache.hadoop.util.ProgramDriver$ProgramDescription.<init>(ProgramDriver.java:59)
> 	at org.apache.hadoop.util.ProgramDriver.addClass(ProgramDriver.java:103)
> 	at org.apache.tez.examples.ExampleDriver.main(ExampleDriver.java:47)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
> 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
> 	at java.lang.reflect.Method.invoke(Method.java:498)
> 	at org.apache.hadoop.util.RunJar.run(RunJar.java:239)
> 	at org.apache.hadoop.util.RunJar.main(RunJar.java:153)
> Caused by: java.lang.ClassNotFoundException: org.apache.tez.dag.api.TezConfiguration
> 	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
> 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
> 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
> 	... 14 more

--- a/tez/README.MD
+++ b/tez/README.MD
@@ -14,6 +14,13 @@ You can use this initialization action to create a new Dataproc cluster with Apa
     gcloud dataproc clusters create <CLUSTER_NAME> \
       --initialization-actions gs://dataproc-initialization-actions/tez/tez.sh
     ```
+1. On Dataproc 1.3+ clusters in order to use pre-installed Tez is necessary to add the flag `--properties 'hadoop-env:HADOOP_CLASSPATH=${HADOOP_CLASSPATH}:/etc/tez/conf:/usr/lib/tez/*:/usr/lib/tez/lib/*'`.
+
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+      --image-version 1.3 \
+      --properties 'hadoop-env:HADOOP_CLASSPATH=${HADOOP_CLASSPATH}:/etc/tez/conf:/usr/lib/tez/*:/usr/lib/tez/lib/*'
+    ```
 1. Hive is be configured to use Tez, rather than MapReduce, as its execution engine. This can significantly speed up some Hive queries. Read more in the [Hive on Tez documentation](https://cwiki.apache.org/confluence/display/Hive/Hive+on+Tez).
 
 ## Tez Web UI

